File: youqugou-web-manager/src/main/webapp/js/service/typeTemplateAuditService.js
/**
 * 贾运通 2018/12/31
 */
app.service("typeTemplateAuditService", function ($http) {
    this.searchTypeTemplateAudit = function (currentPage, itemsPerPage, searchEntity) {
        return $http.post("../typeTemplateAudit/searchTypeTemplateAudit.do?pageNum=" + currentPage
            + "&pageSize=" + itemsPerPage, searchEntity);
    }


    //状态修改【审核】
    this.updateStatus = function (ids,status) {
        return $http.get('../typeTemplateAudit/updateStatus.do?ids='+ids+"&status="+status);
    }

});

File: youqugou-interface/src/main/java/cn/itcast/core/service/TypeTemplateAuditService.java
package cn.itcast.core.service;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.template.TypeTemplate;

public interface TypeTemplateAuditService {
    PageResult searchTypeTemplateAudit(Integer pageNum, Integer pageSize, TypeTemplate typeTemplate);

    void updateStatus(Long[] ids, String status);
}
